src/index.scss

```diff
@@ -1,4 +1,5 @@
 @import '~hds-core/lib/base.css';
+@import '~hds-core/lib/components/loading-spinner/loading-spinner.css' ;
```

Comment:
Are we required to separately import loading-spinner styles from hds? One could expect they come automatically with the component?

Reply:
I'll remove this